<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_replay">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>323</width>
      <height>243</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>320.000000000000000</width>
      <height>240.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_depth">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>323</width>
      <height>243</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>320.000000000000000</width>
      <height>240.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_timer">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>580</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 3px; 
 }
QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 5px;
 } </string>
    </property>
    <property name="title">
     <string>ViewOptions</string>
    </property>
    <widget class="QCheckBox" name="checkBox_normal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_descriptor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Descriptor</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_segment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Segment</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_view_object">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Object</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_resetView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetView</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_test">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>730</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>UnitTest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_virtual_depth">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Virtual Depth</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dataset">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_focus_up">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_focus_down">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>227</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>do</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_focus_left">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>le</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_focus_right">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ri</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
